eth/contracts/DFInitialize.sol

```diff
@@ -116,15 +113,13 @@ contract DFInitialize is WithStorage {
         string memory artifactBaseURI,
         InitArgs memory initArgs
     ) external {
-        // adding ERC165 data
-        LibDiamond.DiamondStorage storage ds = LibDiamond.diamondStorage();
-        ds.supportedInterfaces[type(IERC165).interfaceId] = true;
-        ds.supportedInterfaces[type(IDiamondCut).interfaceId] = true;
-        ds.supportedInterfaces[type(IDiamondLoupe).interfaceId] = true;
-        ds.supportedInterfaces[type(IERC173).interfaceId] = true;
-        ds.supportedInterfaces[type(IERC721).interfaceId] = true;
-        ds.supportedInterfaces[type(IERC721Metadata).interfaceId] = true;
-        ds.supportedInterfaces[type(IERC721Enumerable).interfaceId] = true;
+        ERC165Storage.Layout storage erc165 = ERC165Storage.layout();
+
+        // adding ERC165 data but just for ERC721 because SolidStateDiamond.sol
+        // already has IDiamondReadable, IDiamondWritable, ERC165, and ERC173
+        erc165.setSupportedInterface(type(IERC721).interfaceId, true);
+        erc165.setSupportedInterface(type(IERC721Metadata).interfaceId, true);
+        erc165.setSupportedInterface(type(IERC721Enumerable).interfaceId, true);
```

Comment:
Do we actually want to set these here or is that something we'd want to do in the constructor? I suppose it would be weird/wrong to register these as supportedInterfaces **before** we actually cut the artifact facet 🤔